Java/src/com/couchbase/litecore/C4Document.java

```diff
@@ -275,7 +275,7 @@ static native void resolveConflict(long doc,
 
     // - Purging and Expiration
 
-    static native void setExpiration(long db, String docID, long timestamp)
+    static native boolean setExpiration(long db, String docID, long timestamp)
```

Comment:
The signature should be `static native void setExpiration(long db, String docID, long expiration)`